activitysim/abm/models/util/trip.py

```diff
@@ -123,3 +126,117 @@ def get_time_windows(residual, level):
         else:
             return np.arange(residual + 1)
     return np.concatenate(ranges, axis=1)
+
+
+def initialize_from_tours(tours, addtl_tour_cols_to_preserve=None):
+    """
+    Instantiates a trips table based on tour-level attributes: stop frequency,
+    tour origin, tour destination.
+    """
+
+    stop_frequency_alts = pd.read_csv(
```

Comment:
Fixed. But for my own benefit, can you explain what this change achieves?

Reply:
Consistency across the platform in terms how variables are "injected". It doesn't change the overall functionality, but helps minimize different approaches to solving the same problem across the code. Effectively, make it easier to understand and future developers can assume the setup rather than having to check and make sure.